guideline-app/index.production.ejs

```diff
@@ -4,6 +4,13 @@
     <meta charset="utf-8">
     <title>Designsystemet</title>
     <meta name="viewport" content="width=device-width">
+    <!-- Google Tag Manager -->
```

Comment:
@Lillebo: legger denne saken på `aip=1`-parameteret for oss, eller er det noe vi må gjøre selv? Ser det gjøres på nav.no, antageligvis er det noe som også bør settes i denne løsningen.

Ref:
https://support.google.com/analytics/answer/2763052?hl=no

Reply:
Hm, lurer på om det er noe Tov Are enkelt kan skru på i kontrollpanelet sitt. Kan høre med ham.

Reply:
@erlendev Sjekket med Tov Are: 
>Ja, vi anonymiserer IP som standard i alle målinger.

Så det skal være håndtert 👍